docs/getting-started/Tachyons.stories.mdx

```diff
@@ -91,15 +91,15 @@ Since Tachyons original colors have been replaced with [Orbit Design colors](?pa
 
 The following classes are available:
 
-- `b--[color-name]-[50-900]` (ex. `b--sunray-50`)
-- `hover-b--[color-name]-[50-900]` (ex. `hover-b--sunray-50`)
-- `focus-b--[color-name]-[50-900]` (ex. `focus-b--sunray-50`)
+- `b--[color-name]-[50-900]` (ex. `b--o-ui-sunray-50`)
+- `hover-b--[color-name]-[50-900]` (ex. `hover-b--o-ui-sunrayunray-50`)
```

Comment:
Why add "o-ui" on Tachyons classes?

I mean we did it on css props because someone could use that one but those classes are defined in an atomic css style I dont see why an app would define those.

It also get very long and kindof unpleasant to use IMO

Reply:
bad find and replace